classes/ClaytonTestClass2.cls

```diff
@@ -0,0 +1,322 @@
+		i=3;
+	}
+	public void methodL(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBA(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBB(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBC(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBD(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBE(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBF(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBG(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBH(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBI(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBL(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCA(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCB(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCC(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCD(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCE(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCF(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCG(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCH(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCI(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCL(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDA(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDB(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDC(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDD(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDE(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDF(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDG(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDH(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDI(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDL(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodEA(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodEB(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodEC(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodED(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodEE(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodEF(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodEG(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodEH(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodEI(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodEL(){
```

Comment:
Methods should use the `lowerCamelCase` naming convention. Please rename accordingly.